src/main/java/io/github/cottonmc/cotton/gui/widget/WTiledSprite.java

```diff
@@ -0,0 +1,55 @@
+package io.github.cottonmc.cotton.gui.widget;
+
+import io.github.cottonmc.cotton.gui.client.ScreenDrawing;
+import net.minecraft.util.Identifier;
+
+// It's a Sprite that can be tiled, but does not support animations
```

Comment:
```suggestion
/**
 * A sprite whose texture can be tiled. Tiled sprites don't support animations.
 *
 * @since 2.0.0
 */
```

Reply:
it actually should support animations. i forgot that comment there.

Reply:
Ah, okay. Edited my suggestion.

Reply:
but i guess there should a constructor to pass multiple frames, right?

Reply:
Yep, that would be good.

Reply:
should be fine now a6c26d0bdc71747cdba2f95626a1f2ba8f91db89

please take a look at the new constructor